### ISSUE TITLE
Desktop build translations

### ISSUE THREAD

**Issue by mark-prins** [msupply-foundation/open-msupply#4861]
## What went wrong? 😲

Two issues with translations in desktop
1. The documentation link isn't directing users to the translated version of the docs site, when using a supported language (`es` & `fr`)
2. The menu elements aren't translated

## Expected behaviour 🤔
Redirect the Help > Documentation to the language which the user has selected for the site, if docs are translated in that version

Provide translations for the menu

This will require some investigation - as the menu is constructed in electron prior to loading the site. We would need to use some interaction between the site and electron, as well as dynamically updating menu elements, which I don't know is possible

## How to Reproduce 🔨

Steps to reproduce the behaviour:

1. Run the desktop app
2. Change the language of the site
3. 🤷 restart desktop app?

## Your environment 🌱

- Open mSupply Version:
- Legacy mSupply Central Server Version:

- Platform:

- Database type:

**Comment by alainsussol** [comment]
# 🧪 Testing



- [ ] Start desktop app
- [ ] You can right click to get a menu
- [ ] Change the system language of your device and restart (for now would only work to french - though you can test with an unsupported language, that it falls back to english correctly!)
- [ ] Menus (right click and app menu) are in the correct language

**Comment by alainsussol** [comment]
![Image](https://github.com/user-attachments/assets/785e9eb5-408d-4a88-a8b7-a2d624d3627f)

Only this to remember to translate

**Comment by alainsussol** [comment]
@lache-melvin 

Hi Laché,

These traduction comes from Windows language pack, right ? They are not on Weblate.

**Comment by mark-prins** [comment]
Hi @alainsussol - I've added to weblate now 😄 They are under the 'desktop' project rather than common ( these ones are in a separate file )

**Comment by lache-melvin** [comment]
Yes! Though the exit one I actually missed - in PR now, should be available in weblate later today 😁

**Comment by alainsussol** [comment]
@mark-prins @lache-melvin Done !!!
